public_repo.md

```diff
@@ -0,0 +1,3 @@
+# All Packages available in our public repository
+
+[Package Table](https://c107-224.cloud.gwdg.de/package_table_md.md ':include')
```

Comment:
It would be fantastic to have more meaningful description fields with some spatiotemporal context to get a very quick overview with <kbd>Ctrl</kbd> + <kbd>F</kbd>. Maybe something for a future spring cleaning session.

Reply:
Yes, I agree. But I'd anyway like a better an interactive table, with custom search and filter bars and stuff. Easy to do in principle, just need proper Javascript and a good library to visualise tabular data.